5.5/s2i/bin/run

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Export the Cgroup limits
+set -e
+export_vars=$(cgroup-limits) ; export $export_vars
+set +e
```

Comment:
@mnagy i was following your example in cgroup-limits script ;-) but agree, I prefer || exit 1